api/managementpb/backup/common.proto

```diff
@@ -18,3 +21,25 @@ enum BackupMode {
   INCREMENTAL = 2;
   PITR = 3;
 }
+
+// File represents file or folder on a storage.
+message File {
+  string name = 1 [(validate.rules).string.min_len = 1];
+  bool is_directory = 2;
+}
+
+// BackupToolData contains additional data for backup tools.
+message BackupToolData {
+  // Name of backup in backup tool representation.
+  string name = 1;
+}
```

Comment:
I wonder if we can inline it in Metadata? What is the reason to have it as separate message if it's still generic?

Reply:
We cannot cuz we're passing it separately on restoring.